gr-utils/modtool/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Free Software Foundation, Inc.
+# Copyright 2020 Free Software Foundation, Inc.
```

Comment:
Typically when a file gets changed, the copyright year should be appended, e.g. 2018,2020

Reply:
Yeah, I realized, it's correct in the other files but I missed this one.
It should be fixed now.